protobuf/flow/entities/execution_result.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package flow.entities;
+
+option go_package = "github.com/onflow/flow/protobuf/go/flow/entities";
+option java_package = "org.onflow.protobuf.entities";
+
+message ExecutionResult {
+  bytes previous_result_id = 1;
+  bytes block_id = 2;
+  repeated Chunk chunks = 3;
+  repeated ServiceEvent service_events = 4;
+}
+
+message Chunk {
+  uint32 CollectionIndex = 1;
+  bytes start_state = 2; // state when starting executing this chunk
+  bytes event_collection = 3; // Events generated by executing results
```

Comment:
are these different from these events https://docs.onflow.org/access-api/#event?

Reply:
No, this is hash of the events emitted

Reply:
got it